test/e2e-test/pages/home-office-details/home-office-letter-sent.ts

```diff
@@ -0,0 +1,19 @@
+import { paths } from '../../../../app/paths';
+const config = require('config');
+const { fillInDate } = require('../helper-functions');
+const cache = require('memory-cache');
+
+const testUrl = config.get('testUrl');
+
+module.exports = {
+  homeOfficeLetterSent(I) {
+    Given('I am on the home office letter sent page', async () => {
+      I.amOnPage(testUrl + paths.homeOffice.letterSent);
```

Comment:
Shall we make sure there is no error when navigating to a page? Something like checking page title?